src/OrbitGl/App.cpp

```diff
@@ -1222,6 +1222,13 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> OrbitApp::LoadCaptureFro
   return load_future;
 }
 
+orbit_base::Future<ErrorMessageOr<void>> OrbitApp::MoveCaptureFile(std::filesystem::path& src,
+                                                                   std::filesystem::path& dest) {
+  return thread_pool_->Schedule([src, dest]() { return orbit_base::MoveFile(src, dest); })
```

Comment:
apparently on linux if I try to move the file to a network drive I get errno=18: "Invalid cross-device link" error.
I am kind of ok with this. But we might decide to do copy and remove. also I am not sure if on linux std::fs::rename works the same way. I will try to resurrect my windows build later today and check.

Reply:
moving to another drive works fine on windows